hydra-core/app/controllers/concerns/hydra/controller/search_builder.rb

```diff
@@ -1,9 +1,8 @@
 module Hydra::Controller::SearchBuilder
   extend ActiveSupport::Concern
 
-  # Override blacklight to produce a search_builder that has the current ability in context
-  def search_builder processor_chain = search_params_logic
-    super.tap { |builder| builder.current_ability = current_ability }
+  included do
+    Deprecation.warn Hydra::Controller::SearchBuilder, "Hydra::Controller::SearchBuilder no longer does anything.  It will be removed in Hydra version 10."
```

Comment:
The method was moved into Blacklight::AccessControls::Catalog module (in blacklight-access_controls gem) and will be included into the CatalogController by the generator.  I don't think there is anything that a developer would need to do, but I'll add a note to the deprecation message about where that method was moved to, to make it easier to find if someone needs to know.